onnxruntime/core/providers/dml/DmlExecutionProvider/src/FusedGraphKernel.cpp

```diff
@@ -45,14 +47,17 @@
                 ORT_THROW_IF_FAILED(providerExecutionObject.As(&m_winmlProvider));
             }
 
-            TranslateAndCompileGraph(kernelInfo, graph, node.InputDefs(), node.OutputDefs(), graphNodePropertyMap, transferredInitializerMap);
+            TranslateAndCompileGraph(kernelInfo, graph, node.InputDefs(), fusedNodeInputArgOriginalNames, 
+                node.OutputDefs(), fusedNodeOutputArgOriginalNames, graphNodePropertyMap, transferredInitializerMap);
         }
 
         void TranslateAndCompileGraph(
             const onnxruntime::OpKernelInfo& kernelInfo,
             const onnxruntime::Graph& graph,
             const onnxruntime::ConstPointerContainer<std::vector<onnxruntime::NodeArg*>>& fusedNodeInputDefs,
+            std::vector<std::string>& fusedNodeInputArgOriginalNames,
```

Comment:
>std::vector<std::string>& [](http://example.com/codeflow?start=12&length=25)

Are these mutable? If not, add `const`. Also consider whether the interface is specific to vector (do we ever resize it or call vector-specific methods?), or whether any contiguous series of elements (`std::array`, `std::vector`...) is acceptable via `gsl::span`.

See ExecuteOperator.

```
        void ExecuteOperator(
            IDMLCompiledOperator* op,
            _In_opt_ const DML_BUFFER_BINDING* persistentResourceBinding,
            gsl::span<ID3D12Resource*> inputTensors,
            gsl::span<IMLOperatorTensor*> outputTensors) const 
```